relay/robot.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+import wpilib
+
+
+class MyRobot(wpilib.TimedRobot):
+    """
+    This is a sample program which uses joystick buttons to control a relay.
+    A Relay (generally a spike) has two outputs, each of which can be at either
+    0V or 12V and so can be used for actions such as turning a motor off, full
+    forwards, or full reverse, and is generally used on the compressor. This
+    program uses two buttons on a joystick and each button corresponds to one
+    output; pressing the button sets the output to 12V and releasing sets it to 0V.
+    """
+
+    def robotInit(self):
+        """Robot initialization function"""
+
+        self.relay = wpilib.Relay(0)
+
+        self.joystickChannel = 0
+        self.joystick = wpilib.Joystick(self.joystickChannel)
```

Comment:
The Java/C++ example doesn't define a separate variable for the joystick channel number; let's avoid introducing one here.
```suggestion
        self.joystick = wpilib.Joystick(0)
```